tests/test_grid3d/test_grid_properties.py

```diff
@@ -52,6 +52,21 @@ def test_import_init():
     assert poro.values.mean() == pytest.approx(0.1677402, abs=0.00001)
 
 
+def test_gridproperties_iter():
+    g = Grid()
+    g.from_file(GFILE1, fformat="egrid")
```

Comment:
Can you change this to:
```suggestion
xtgeo.grid_from_file(GFILE1, fformat="egrid")

Reply:
Did you look into this?

Reply:
I added a separate commit for removing all `Grid().from_file` from that file.